### ISSUE TITLE
Make notification emails more personalized

### ISSUE THREAD

**Issue by joatu** [joatuapp/joatu-v2#122]
Right now they come from a 'noreply' address and I've had some criticism on that and was referenced to this link: http://emailcritic.com/2013/01/down-with-do-not-reply-emails/

Can we make them more inviting?  Even if they looked as bad as facebook email notification emails look, it would be a step in the right direction.

**Comment by undergroundwebdesigns** [comment]
We can certainly make the emails more inviting. The easiest way to do this is to make a mailchimp / mandril template, and link it to an issue (like this one)... it's fairly easy for me to switch the code over to using those templates for any email it sends. And I can pass through whatever variables are appropriate (message subjects, recipient names, etc).

Regarding the 'noreply' address.. granted, I only skimmed the first bit of the article, but they seem to be missing a fundamental point: "noreply" addresses are, by convention, used by computer programs. Websites. Not people. That's how we're using them as well. The email notification you receive saying "you've been invited to JoatU" ultimately comes from a computer program. Replying to the computer program makes no sense.

We _should_ have a link in the email to how they can contact JoatU. But "replying" to automated emails is not the way to do it.

**Comment by joatu** [comment]
Why do notification emails from facebook say that they're from facebook and have this strange email "notification+feorf1h1@facebookmail.com" whereas our emails show up from the noreply email and don't have a name associated simply saying "JoatU"?  I see them as being from "no-reply" not being from "JoatU".  I think that's the first big step.  

And in the title, instead of "Mailboxer new reply" for it to say "JoatU new reply"

P.S. When he sent that article, I failed to mention that he was referring specifically to the list in the middle of it:

"The best email marketing gets delivered, opened, acted upon. It also helps to build brand and goodwill. A do-not-reply email simply can’t do those last two for you. Consider the company that sends a real email instead of do-not-reply. By doing so, they are subconsciously telling customers:
We are real people at the other end of this email.
We recognize that you are real people on the receiving end of this email.
We do want to hear from you.
We want to make it easy for you to do business with us.
We want to save you time and hassle by letting you immediately reply to this email."

**Comment by undergroundwebdesigns** [comment]
The "from name" thing actually has nothing to do with the actually email used, and we can set it to whatever we want. It just defaults to the part before the @, if nothing else is specified, which is why you're seeing 'no-reply' right now. As for FB's strange emails.... I don't know. There's any number of things they could be using that for. Note that, as with google, the part after the + doesn't mean anything (from an email routing perspective), so the email is really 'notification@facebookmail.com', which is, if anything, _worse_ than a no-reply, as it's not clear whether it's an automated message or not.

We can also update the title, content, etc. as I said: best way to do that is with Mandril or Mailchimp.

And back to the article, again, his main point is "The best email marketing....." notification emails are not marketing. They're not supposed to seem like they came from a human. We're not pretending they came from a person, but making it hard to reply... we're just saying that the system is notifying you of something you may be interested in.

**Comment by joatu** [comment]
We're hand-delivering a human-made message.  You're right that the email didn't come from a human, but the initial message that's included did.  Anyhow, I'm in Mandrill but don't see any options of how to 

- [ ] name the "no-reply" emails so they say they're from "JoatU"
- [ ] change "Mailboxer new message" and "Mailboxer new reply" to "JoatU new message" and "JoatU new reply"

All the other changes related to these emails can be delayed for awhile.

**Comment by undergroundwebdesigns** [comment]
oh, sorry. Changing the name & subject are best done in the code, if that's all we're changing. If you were creating a full template, there's a "subject" field you can use to set the subject.

If you're talking strictly about messages between members, and notifications about those messages, then yes, things are a little murkier. However, what should we put? If we put the sender's email address, then a) that might be a breach of privacy, and b) we're encouraging people to email directly, and _not_ use JoatU for their messages. If we put any @joatu.com address, and people respond, either they'll be expecting us to get it? or else somehow thinking their response will go to the person who sent them the message, which it won't. We _could_ eventually set up a feature such that replying to those messages _does_ serve to reply to the conversation in JoatU, but setting that up is a project in its own right, and may incur costs (from Mandril, would have to check their pricing on emails received).

**Comment by joatu** [comment]
Just putting that it's from JoatU (in the same way facebook emails work) is fine and normal.  That's how just about every other site does it.  We don't want to breach the privacy.  And we don't need to have people be able to send emails through the email (because of potential costs).

It's fine that the email remains noreply@joatu.com it would just be better if that email had the name "JoatU" attached to it.

**Comment by adriamarcet** [comment]
So it seems like a Pandora's box here... 

@undergroundwebdesigns  you're right, this is not marketing purpouse but it's a comunication coming from a platform wich is intended to share skills and human knowledge. I suggested changing the noreply@joatu.com address to something more 'human-like' as I think the platform should reflect. But in the other hand you can't tell people to reply to that address so why don't you try to have one like: automaticmessage@joatu.org. Also, inside it specify something like : "Remember that this email is unattended. Joatu has generated this message automaticly for you. Even so, you can send us your thoughts whenever you want at this addres: hello@joatu.org. Let's keep in contact. :)

Same here. :)

**Comment by undergroundwebdesigns** [comment]
I completely agree with adding a message like "Remember this email is unattended... " as you suggested, but I'd still rather not change the "noreply" email. I don't think "automaticmessge" is any friendlier than "noreply", and it's also confusing. People understand that "noreply" messages mean automated emails. If we use "automaticmessage" or something else, people will probably still figure it out, but it'll take some extra effort on their part, why not go with the unofficial "standard" and make it easy for people?

**Comment by rsmithlal** [comment]
What's the status of this issue? How do things currently stand?

**Comment by joatu** [comment]
@rsmithlal  My understanding is that the code I created in mailchimp was copied somewhere (mandrill?) and would need to be modified there to update the automatic emails that are generated and sent to users.

**Comment by rsmithlal** [comment]
Was this specifically about the from address, or the subject, or the email template? Do you think it might be a good idea to close this issue and create a new one with the concerns more relevant to today? It's a little hard to follow the original conversation.

**Comment by joatu** [comment]
We can close this issue and refer to #187
